samples/apps/smallbank/smallbank.cmake

```diff
@@ -16,45 +16,46 @@ sign_app_library(
 
 if(BUILD_TESTS)
   # Small Bank end to end and performance test
-  if(PBFT)
-    if(NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
-      set(SMALL_BANK_VERIFICATION_FILE
-          ${CMAKE_CURRENT_LIST_DIR}/tests/verify_small_bank_50k.json
-      )
-      set(SMALL_BANK_ITERATIONS 50000)
+  foreach(CONSENSUS ${CONSENSUSES})
+
+    if(${CONSENSUS} STREQUAL "pbft")
+      if(NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
+        set(SMALL_BANK_VERIFICATION_FILE
+            ${CMAKE_CURRENT_LIST_DIR}/tests/verify_small_bank_50k.json
+        )
+        set(SMALL_BANK_ITERATIONS 50000)
+      else()
+        set(SMALL_BANK_VERIFICATION_FILE
+            ${CMAKE_CURRENT_LIST_DIR}/tests/verify_small_bank_2k.json
+        )
+        set(SMALL_BANK_ITERATIONS 2000)
+      endif()
     else()
       set(SMALL_BANK_VERIFICATION_FILE
-          ${CMAKE_CURRENT_LIST_DIR}/tests/verify_small_bank_2k.json
+          ${CMAKE_CURRENT_LIST_DIR}/tests/verify_small_bank.json
       )
-      set(SMALL_BANK_ITERATIONS 2000)
+      set(SMALL_BANK_ITERATIONS 200000)
     endif()
-  else()
-    set(SMALL_BANK_VERIFICATION_FILE
-        ${CMAKE_CURRENT_LIST_DIR}/tests/verify_small_bank.json
+
+    add_perf_test(
+      NAME small_bank_client_test_${CONSENSUS}
+      PYTHON_SCRIPT ${CMAKE_CURRENT_LIST_DIR}/tests/small_bank_client.py
+      CLIENT_BIN ./small_bank_client
+      VERIFICATION_FILE ${SMALL_BANK_VERIFICATION_FILE}
+      LABEL SB
```

Comment:
We're reusing this label for multiple builds. I think this should be `SB_${CONSENSUS}` so that they look different to metrics (and to whatever else consumes this label).